app/Models/Description.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Description extends Model
+{
+    /**
+    * Get the task that owns the description
+    */
+    public function task()
+    {
+        return $this->belongsTo('Task::class');
```

Comment:
```suggestion
        return $this->belongsTo(Task::class);
```
Viết như này nhé, như kia nó là string rồi